modules/ROOT/pages/gatekeeper.adoc

```diff
@@ -4,21 +4,17 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: runtime, gateway, gatekeeper, security, policy, policies
 
-When you deploy an application, there could be a lapse during which the tracked resources exposed by the application are accessible, but the policies specified for those tracked resources are not yet applied, so unwelcome traffic may reach the them. GateKeeper prevents this situation by blocking the tracked resource until all policies have been retrieved and applied without errors. During this period, the API returns a 503 HTTP status code.
+When you deploy an application, the tracked resources exposed by the application might become accessible due to a lapse. Because the policies specified for those tracked resources are not yet applied, the API might receive unwanted traffic. GateKeeper prevents this situation by blocking the tracked resource until all policies are retrieved and applied without errors. 
 
 == How Gatekeeper Works
 
-The Gatekeeper mechanism only applies when (re)starting the runtime or (re)deploying a tracked resource (ie one linked with an API instance through Autodiscovery).
+The Gatekeeper mechanism only applies when you (re)starting the runtime or (re)deploying a tracked resource that is linked with an API instance through Autodiscovery. When Gatekeeper is engaged, each tracked resource is blocked ("503 - Service Unavailable" HTTP status code) until all policies that are applied to the API instance in API Manager are run without errors.
```

Comment:
you starting seems wrong

Reply:
API Manager are run without errors ==> API Manager are applied without errors

Reply:
Done! Thanks!